### PR TITLE
Add CEReactions tests for HTMLBaseElement

### PR DESCRIPTION
**Pull request by xiuqijix**
part of issue #5727

### CODE REVIEW

**Comment by kaixinjxq** [comment]
@rniwa, would you like to help to review this PR?

File: custom-elements/reactions/HTMLBaseElement.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/custom-elements-helpers.js"></script>
+</head>
+<script>
+
+testReflectAttribute('href', 'href', '/', '/', 'http://example.com/', 'http://example.com/', 'href on HTMLBaseElement', 'base', HTMLBaseElement);
+testReflectAttribute('target', 'target', '_blank', '_blank', '_self', '_self', 'target on HTMLBaseElement', 'base', HTMLBaseElement);
+
+function testReflectAttribute(jsAtName, coAtName, jsAtValue1, coAtValue1, jsAtValue2, coAtValue2, name, elementName, interfaceName) {
```
Comment (rniwa) [inline custom-elements/reactions/HTMLBaseElement.html]: This function should probably be added to custom-elements-helpers.js instead.

Comment (kaixinjxq) [reply custom-elements/reactions/HTMLBaseElement.html]: @rniwa, many thanks for your review, I think [reactions.js](https://github.com/web-platform-tests/wpt/blob/master/custom-elements/reactions/resources/reactions.js) is more appropriate, and I've addressed it in latest commit and replaced the `testReflectAttrWithParentNode` function in [HTMLButtonElement.html](https://github.com/web-platform-tests/wpt/blob/master/custom-elements/reactions/HTMLButtonElement.html) as well, PTAL.

**Comment by foolip** [comment]
This PR is blocked on the required "Travis CI - Pull Request" check after https://github.com/web-platform-tests/wpt/issues/14499. In order to trigger it, I will close and reopen this PR.

**Comment by kaixinjxq** [comment]
@rniwa, I'm working on another `CEReactions` tests that may base on this new common function, could you please check if this PR is fine to you?

**Comment by kaixinjxq** [comment]
@rniwa, friendly ping.

**Review by domenic — APPROVED** [review]

**Comment by domenic** [comment]
LGTM but needs rebasing. It may also be nicer to do separate PRs to fix existing tests, instead of putting that work in this PR.

**Comment by kaixinjxq** [comment]
@domenic, many thanks for your review! I just rebase this PR and revert the commit "Modify the tests for boolean attributes", I will address it in a separate PR.

**Pull request merged**